front/src/applications/operationalStudies/utils.ts

```diff
@@ -155,35 +158,85 @@ export const formatElectrificationRanges = (
     };
   });
 
+/**
+ * Check if the train path used waypoints added by map click and add them to the operational points
+ */
+export const upsertMapWaypointsInOperationalPoints = (
+  trainschedulePath: TrainScheduleResult['path'],
```

Comment:
- you forgot a S maj in trainschedule ;) 
- maybe `path` is enough (only a suggestion)
```suggestion
  trainSchedulePath: TrainScheduleResult['path'],
```

Reply:
Fine by me for `path` :) Done.